dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -26,7 +26,7 @@ debs/{{ deb }}{{' '}}
 RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
 RUN rm -rf /debs
 
-COPY ["start.sh", "orchagent.sh", "/usr/bin/"]
+COPY ["start.sh", "orchagent.sh", "swssconfig.sh", "/usr/bin/"]
```

Comment:
to rename it just to config.sh? swssconfig.sh will be confused with swssconfig script.